samples/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,55 @@
+# Using Microsoft XmlSerializer Generator on .Net Core
+
+When a running application creates an instance of XmlSerializer, it generates a temporary assembly containing the implementation of the serializer. This happens once for each type for which an XmlSerializer is needed. To improve performance, we can use Microsoft XmlSerializer Generator to pre-generate code for an XmlSerializer for each specified type in a project and compile it into an assembly. This assembly will then be used by the application when creating an XmlSerializer instead of generating a temporary one.
+
+## Prerequisition
+
+You need to install the followings on your machine:
+* [.NET Core SDK daily builds](https://github.com/dotnet/cli#installers-and-binaries) 
+* [.NET Core runtime daily builds](https://github.com/dotnet/core-setup#daily-builds). 
+
+You can validate your .NET Core SDK version by typing `dotnet --info`.
+
+## Instructions
+
+Here are the step by step instructions how to create a dotnet project and install the XmlSerializer Generator for that project.
+
+1. Create a library project that contains your custom data types. E.g. create a library named MyData using the CLI: `dotnet new library --name MyData`
+
+2. Go to the folder of the project you created in step 1, e.g. `cd MyData`
+
+3.  Add a nuget.config file in the root of the project, using the following:
```

Comment:
This step is a bit confusing until I read everything and understood what is for. How about we explain the purpose at the beginning? For example, change this line to "Add dotnet-core MyGet feed to the project. This step won't be necessary when this tool is released to NuGet.org eventually."

Reply:
You can point to this nuget.config, which might be easier for folks: https://github.com/dotnet/core/blob/master/samples/nuget.config

Reply:
Yeah, already include this link here.